src/constants.rs

```diff
@@ -19,3 +19,4 @@ pub const CHANNEL_LATEST_FILE_NAME: &str = "channel-fuel-latest.toml";
 pub const CHANNEL_NIGHTLY_FILE_NAME: &str = "channel-fuel-nightly.toml";
 
 pub const DATE_FORMAT: &[FormatItem] = format_description!("[year]-[month]-[day]");
+pub const DATE_FORMAT_URL_FRIENDLY: &[FormatItem] = format_description!("[year]/[month]/[day]");
```

Comment:
nit

```suggestion
pub const DATE_FORMAT: &[FormatItem] = format_description!("YYYY-MM-DD");
pub const DATE_FORMAT_URL_FRIENDLY: &[FormatItem] = format_description!("YYYY/MM/DD");
```

Reply:
This is part of the API within the [format_description!](https://docs.rs/time/latest/src/time/format_description/component.rs.html#78) implementation (unless I'm missing something)